source/includes/steps-enable-authentication-in-shardcluster-nodowntime-transition-mongos.yaml

```diff
@@ -50,17 +51,20 @@ pre: |
   method against the ``admin`` database to safely shut down the
   :program:`mongos`.
   
-  Restart one :program:`mongos` at a time to ensure that clients
-  can connect to the sharded cluster. If your cluster has only one
-  :program:`mongos`, this step results in downtime while the :program:`mongos`
-  is offline.
+  Restart one :program:`mongos` at a time to ensure that clients can connect
+  to the sharded cluster. If your cluster has only one :program:`mongos`, this
+  step results in downtime while the :program:`mongos` is offline.
+  
+  Connect to the :program:`mongos` and use the :method:`db.shutdownServer()`
+  method against the ``admin`` database to safely shut down the
+  :program:`mongos`.
   
   .. code-block:: javascript
   
      db.getSiblingDB("admin").shutdownServer()
   
-  You can then start the :program:`mongos` with the new configuration
-  file, specifying the path to the config file using :option:`--config`.
+  Restart :program:`mongos` with the new configuration file, specifying the
+  path to the config file using :option:`--config`.
```

Comment:
I agree that's a nice distinction, but unfortunately, I don't think we're doing that consistently.  Take a look at https://docs-mongodbcom-staging.corp.mongodb.com/rkumar/DOCS-8616/tutorial/enforce-keyfile-access-control-in-existing-sharded-cluster-no-downtime.html#create-the-adminstrator-user  vs  https://docs-mongodbcom-staging.corp.mongodb.com/rkumar/DOCS-8616/tutorial/enforce-keyfile-access-control-in-existing-sharded-cluster-no-downtime.html#optional-create-users-for-client-applications-to-authenticate-with

Other than naming the user in the example sentences, how is the user "admin" that much different than user "joe"?

Even in here, how is it in context if we're specifying the conf file name but not the path?  It's a bit general and specific all jumbled up.